### DISCUSSION TITLE
✅ Snippets - enter carriage

### DISCUSSION THREAD

**Discussion by shikilee** [isontheline/pro.webssh.net#1113] [Q&A] [answered]
Hi All,
I created a simple snippet with a linux simple command: find / -type f -mtime +30 -size +10M -exec ls -lh {} \;
How to add return carriage ? so that I don't need to manually enter from keyboard.

**Comment by isontheline** [comment, marked as answer]
Hi @shikilee 👋 

Just put a carriage return (by pressing the return key) on your snippet command.

**Reply by shikilee** [reply]
🙏 Thank you very much it works so simple and beautifully 😄
